XenAdmin/Wizards/DRWizards/DRFailoverWizardStoragePage.cs

```diff
@@ -332,8 +332,7 @@ private List<FibreChannelDevice> FiberChannelScan()
 
             Host master = Helpers.GetMaster(Connection);
 
-            if (master != null && (Helpers.HostBuildNumber(master) >= 9633
-                                || Helpers.HostBuildNumber(master) == Helpers.CUSTOM_BUILD_NUMBER))
+            if (master != null)

```

Comment:
We don't need the master check anymore and we should remove the `dconf[SrProbeAction.DEVICE] = device.Path;` completelly. That was only applicable for old versions.